src/lib/utils/SynapseClient.ts

```diff
@@ -332,14 +352,14 @@ export const addFilesToDownloadList = (
   updateParentState?: any,
 ) => {
   return doPost<AsyncJobId>(
-    `file/v1/download/list/add/async/start`,
+    `/file/v1/download/list/add/async/start`,
```

Comment:
For all of these added slashes, see https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1079/files#r651002737